apps/dash-label-properties/requirements.txt

```diff
@@ -0,0 +1,16 @@
+datashader==0.8.0
+dash-core-components
```

Comment:
I thought that when you put `dash` in the `requirements.txt`, or pip install dash, you got dash-core-components, dash-html-components and dash-table together with `dash`. Am I missing something? (maybe it's related to the datashader problems?) I'm asking because we've been trying to tell users (on the forum for example) to install `dash` only and not to separately install dcc and html because they would run into version incompatibilities.

Reply:
I may have just copied this over from an existing requirements.txt. I'll make sure we don't need it and then trim the requirements!

Reply:
I looked at pipdeptree and all we really need is `dash`, `dash_canvas` and `dash_bootstrap_components` (now that we got rid of the datashader dependencies). For visibility purposes it might make sense to add `skimage` back in, what do you think @emmanuelle